flask_security/forms.py

```diff
@@ -125,7 +125,8 @@ class NewPasswordFormMixin():
 class PasswordConfirmFormMixin():
     password_confirm = PasswordField(
         get_form_field_label('retype_password'),
-        validators=[EqualTo('password', message='RETYPE_PASSWORD_MISMATCH')])
+        validators=[password_required,
```

Comment:
@jxltom thank you for the clarification. Can you keep the `EqualTo` validator first so user get expected message  `RETYPE_PASSWORD_MISMATCH`?